applications/crossbar/priv/api/swagger.json

```diff
@@ -12702,37 +12702,9 @@
         "version": "2.0"
     },
     "paths": {
-        "/v2/about": {
-            "get": {}
-        },
```

Comment:
It was easiest to code by assuming the API endpoint operated under an account, and then manually update cb_api_endpoints to include logic to override that assumption. There's nothing in the endpoint modules themselves that would easily identify those APIs that operate in an account, without an account, or with either. [This](https://github.com/2600hz/kazoo/pull/2369/files/057c90b0d82dc492918e6da1684301b1da34e136#diff-78a153b2268aba9e7897d49d708b1f26L468) change is why the endpoint above is being removed.